tooling/nargo/src/artifacts/debug_vars.rs

```diff
@@ -0,0 +1,75 @@
+use acvm::acir::brillig::Value;
+use noirc_printable_type::PrintableType;
+use std::collections::{HashMap, HashSet};
+
+#[derive(Debug, Default, Clone)]
+pub struct DebugVars {
+    id_to_name: HashMap<u32, String>,
+    active: HashSet<u32>,
+    id_to_value: HashMap<u32, Value>, // TODO: something more sophisticated for lexical levels
+    id_to_type: HashMap<u32, u32>,
+    types: HashMap<u32, PrintableType>,
+}
+
+impl DebugVars {
+    pub fn new(vars: &HashMap<u32, String>) -> Self {
+        let mut debug_vars = Self::default();
+        debug_vars.id_to_name = vars.clone();
+        debug_vars
+    }
+
+    pub fn get_variables<'a>(&'a self) -> Vec<(&'a str, &'a Value, &'a PrintableType)> {
```

Comment:
```suggestion
    pub fn get_variables(&self) -> Vec<(&str, &Value, &PrintableType)> {
```

Lifetimes aren't necessary here.